project-base/storefront/public/locales/cs/common.json

```diff
@@ -495,6 +495,7 @@
     "You": "Vy",
     "You are being redirected...": "Budete přesměrováni...",
     "You can also buy": "Můžete si dokoupit",
+    "You can check the status on the order detail page.": "Stav můžete zkontrolovat na detailu objednávky",
```

Comment:
Missing dot at the end of the Czech translation.

Reply:
fixed